aspnetcore/blazor/hosting-models.md

```diff
@@ -434,23 +440,8 @@ To set a component's hosting model to Blazor Server or Blazor WebAssembly at com
 
 ## Additional resources
 
-:::moniker range=">= aspnetcore-6.0"
-
-* <xref:blazor/hybrid/index>
-* <xref:blazor/tooling>
-* <xref:blazor/project-structure>
-* <xref:signalr/introduction>
-* <xref:blazor/fundamentals/signalr>
-* <xref:blazor/tutorials/signalr-blazor>
-
-:::moniker-end
-
-:::moniker range="< aspnetcore-6.0"
-
-* <xref:blazor/tooling>
-* <xref:blazor/project-structure>
-* <xref:signalr/introduction>
-* <xref:blazor/fundamentals/signalr>
-* <xref:blazor/tutorials/signalr-blazor>
-
-:::moniker-end
+* [HTML](https://www.w3.org/html/)
+* [WebAssembly](https://webassembly.org)
+* [mono/mono GitHub repository](https://github.com/mono/mono)
+* [C# Guide](/dotnet/csharp/)
+* <xref:mvc/views/razor>
```

Comment:
These additional resources seem a bit random. Where did these come from?

Reply:
***Ancient times!*** 👴😆 ... on the grounds in some cases that devs may arrive *new to programming* and/or *new to anything MS offers*.

Let me know which, if any, or others that you'd like to place.

Reply:
Oh, I see what you mean. Yes, they were moved and recomposed from the original list at the bottom of the Overview. You can see them live here ...

https://learn.microsoft.com/en-us/aspnet/core/blazor/?view=aspnetcore-8.0#additional-resources

Reply:
OK, well, I think we could remove all of them. At the very least, we should remove the link to the mono repo, which is completely irrelevant.